test/XRP/common-xrpl-client.test.ts

```diff
@@ -24,7 +24,7 @@ import {
   LastLedgerSequence,
 } from '../../src/XRP/Generated/node/org/xrpl/rpc/v1/common_pb'
 import { AccountRoot } from '../../src/XRP/Generated/node/org/xrpl/rpc/v1/ledger_objects_pb'
-import TransactionResult from '../../src/XRP/model/transaction-result'
+import TransactionResult from '../../src/XRP/model/final-transaction-result'
```

Comment:
shall we use `FinalTransactionResult` here?

Reply:
yep, I think I updated this... not sure why it's not erroring

Reply:
oh wait maybe I do... cause you get to name the default export whatever you want in typescript?